activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1610,8 +1614,8 @@ def remove_column_for_alter(table_name, column_name, type = nil, **options)
           "DROP COLUMN #{quote_column_name(column_name)}"
         end
 
-        def remove_columns_for_alter(table_name, *column_names, **options)
-          column_names.map { |column_name| remove_column_for_alter(table_name, column_name) }
+        def remove_columns_for_alter(table_name, *column_names, type: nil, **options)
+          column_names.map { |column_name| remove_column_for_alter(table_name, column_name, type, **options) }
```

Comment:
This change seems unnecessary.

Reply:
You're right. I suppose I was trying to unify these signature and pass all args through, but I agree that it is unnecessary to achieve the feature.

I reverted this line in "Revert method signature changes and extra args"